RecoTauTag/Configuration/python/tools/dumpTauVariables_cfg.py

```diff
@@ -0,0 +1,30 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("DUMP")
```

Comment:
@roger-wolf @slava77- I agree w/ @slava77 comment here (and for compareTauVariables.py) - could you have a look at changing this?